### ISSUE TITLE
LRP on PVC and PACE signals

### ISSUE THREAD

**Issue by hoda213** [helme/ecg_ptbxl_benchmarking#13]
Hello,
I was reading your article about LRP with e-rule =0.1 on ECG signals but I can not find it in this repository can you share it with me ?
Thanks

**Comment by nstrodt** [comment]
Dear @hoda213,
thank you for your interest in our work. Currently, we are not planning to release the code for all the experiments in the second part of our paper as they were supposed to illustrate interesting directions for follow-up work rather than complete investigations. Therefore, we decided to restrict also the code repository to the benchmarking part. There are very good libraries that implement various interpretability methods such as Innvestigate (https://github.com/albermax/innvestigate) for keras or Captum (https://captum.ai/) for Pytorch. The latter is currently lacking support for LRP, but similar qualitative insights can be reached with related methods such Integrated Gradients (which is even equivalent to epsilon-LRP under certain conditions).
